src/App.jsx

```diff
@@ -4,9 +4,18 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
 
   render() {
+    const { query } = this.state;
+    const filteredMovies = moviesFromServer
+      .filter(
+        movie => movie.title.toLowerCase().startsWith(query.toLowerCase())
+        || movie.description.toLowerCase().startsWith(query.toLowerCase()),
+      );
```

Comment:
```suggestion
          .filter(
        ({title, description}) => title.toLowerCase().includes(query.toLowerCase())
        || description.toLowerCase().includes(query.toLowerCase()),
      );
```
В задание by searching it, думаю просто надо искать в тексте.